cmd/bot/main.go

```diff
@@ -21,7 +21,8 @@ import (
 )
 
 //go:embed comment.md
-var commentTemplate string
+//nolint:unused
+var commentTemplate string //
```

Comment:
```suggestion
var commentTemplate string
```

?

Reply:
Heh, left the trailing comment as I was initially trying to add the `//nolint:...` directive on the line.